### DISCUSSION TITLE
Evaluator Question: Which algorithm should you use to find a node that is close to the root of the tree?

### DISCUSSION THREAD

**Discussion by mike-goetz** [realAlgoMonster/contrib#1] [General]
The Evaluator Question about "Which algorithm should you use to find a node that is close to the root of the tree?" states that the DFS is the correct answer.

I would disagree since the target is close to the root. Therefore BFS should be used.

If the target is the Node with value 7, DFS would be a bad choice.
![tree](https://user-images.githubusercontent.com/8265259/112736292-f1fdd800-8f51-11eb-9edc-8f388a7b5ff2.png)

**Comment by realAlgoMonster** [comment]
Thanks for reporting. You are right. Fixed.
